paper.tex

```diff
@@ -2125,6 +2161,158 @@ \section{Editour API Documentation}
 
 \clearpage
 
+\section{Ritsumeikan Heart Project}
+\label{sec:ritsumeikanHeartProject}
+
+Throughout our stay at Ritsumeikan, we also had the opportunity to work closely
+with other Ritsumeikan students to create build a project that involved the
+integration of hardware and software. With the help of students and professors
+alike, we were able to produce an educational physical model of a heart
+controlled by an Arduino microcomputer. Noma gave us tremendous assistance with
```

Comment:
Professor Noma

Reply:
I call him Noma Sensei before so I changed it to that